packages/@aws-cdk/aws-stepfunctions-tasks/lib/codebuild/start-build.ts

```diff
@@ -1,9 +1,122 @@
 import * as codebuild from '@aws-cdk/aws-codebuild';
 import * as iam from '@aws-cdk/aws-iam';
+import * as kms from '@aws-cdk/aws-kms';
 import * as sfn from '@aws-cdk/aws-stepfunctions';
 import * as cdk from '@aws-cdk/core';
 import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
 
+/**
+ * Status for log configuration
+ */
+export enum BuildLogConfigStatus {
```

Comment:
This is the wrong place for this type. I assume that's because `Project` does not support logging. If you want to add this, it needs to be done in the `@aws-cdk/aws-codebuild` module. I understand if this is too complicated in this PR, and in that case just remove it. But this should not be in this module.